WebAPI/Identity/MyIdentityDataInitializer.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Linq;
+using WebAPI.Models.Auth;
+
+namespace WebAPI.Identity
+{
+    public static class MyIdentityDataInitializer
+    {
+        public static void SeedData
+        (UserManager<User> userMenager, RoleManager<IdentityRole<int>> roleMenager, IConfiguration configuration)
+        {
+            SeedRoles(roleMenager);
+            SeedUsers(userMenager, configuration);
+        }
+
+        public static void SeedUsers(UserManager<User> userManager, IConfiguration configuration)
+        {
+            var userExists = userManager.FindByNameAsync("Admin").Result;
+            if (userExists == null)
```

Comment:
`userExists` variable sounds like it is a boolean, so either change its name, or make it boolean by comparing
`var userExists = userManager.FindByNameAsync("Admin").Result != null;`
or
`var userExists = await userManager.FindByNameAsync("Admin") != null;`